packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Server/elements.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import { Button } from '@codesandbox/common/lib/components/Button';
+import PowerIcon from 'react-icons/lib/md/power-settings-new';
+import { WorkspaceInputContainer } from '../../elements';
+
+export const SubTitle = styled.div`
+  text-transform: uppercase;
+  font-weight: 700;
+  color: ${props =>
+    props.theme.light ? 'rgba(0, 0, 0, 0.6)' : 'rgba(255, 255, 255, 0.6)'};
+
+  padding-left: 1rem;
+  font-size: 0.875rem;
+`;
+
+export const TasksContainer = styled(WorkspaceInputContainer)<{
+  disconnected: boolean;
+}>`
+  display: flex;
+  flex-direction: column;
+  pointer-events: ${props => (props.disconnected ? 'none' : 'initial')};
```

Comment:
I think it's better to use @Saeris' method of only having 1 time the props (destructur).

```ts
export const TasksContainer = styled(WorkspaceInputContainer)<{
  disconnected: boolean;
}>`
  ${({ disconnected }) => css`
    // ...
    pointer-events: `${disconnected ? 'none' : 'initial'}`;
    // ... 
  `};
`;
```

Reply:
Oh man, I don't like that syntax I think it makes waaay more confusing

Reply:
But that's the syntax we agreed on starting to use, no? 🤔

Reply:
I don't think so 🤔

Reply:
I will ask in the chat

Reply:
So there isn't really a consensus here I don't think. IMO it's preferable to do one destructure so that it's easier to extend a stylesheet with more variables later on and see in one place all the individual properties being used. By accessing them off of `props`, eslint isn't able to tell you when a property is unused, and in the case where you use a property multiple times in a single stylesheet, you can quickly lose track of the usage. Additionally it cuts down on overall verbosity in the code, because you're not repeatedly accessing off of `props`.

Reply:
It was in @Saeris' PR where he did the big TypeScript refactor of a lot of components (#2066).

> - For styled components, Inline types are fine where their footprint is small, ie:
```ts
const SomeElement = styled.div<{ large: boolean }>`
  ${({ large }) => css`
    /* 
      Always wrap style declarations in a single function call and don't forget to
      use "css" for syntax highlighting!
    */
  `}
`
```